net/atftp/files/atftpd.init

```diff
@@ -18,9 +19,11 @@ start_service() {
 
 	config_get srv service path "/srv/tftp"
 	config_get port service port 69
+	config_get mdns service mdns 1
```

Comment:
Please do not set mdns to `1` what means `enabled`.
The default behavior should be `0` therefore `disabled`.

The behavior should not change

Reply:
Hello @feckert,

Apologies for the very late response.

Set to 0 by default.

Kindly review.